File: 2/thanos.cpp
#include <iostream>
#include <conio.h>
using namespace std;
int main(){
int penduduk = 1;
for(int i = 1 ; i <= 50 ; i++){
    if(i==1){

    }
    else if(i%3==0){
        penduduk /= 2;
    }else{
        penduduk *= 3;
    }
    cout<<penduduk<<endl;
}

}

File: 2/thanos.js
let population = 1;
for (let day = 1; day <= 49; day++) {
  if (day % 3 === 0) {
    population = Math.floor(population / 2);
  } else {
    population = population * 3;
  }
  document.write("Jumlah Penduduk hari ke- "+day);
}
